### PR TITLE
Tapping the 'Movie' in 'MovieSearchViewController'.

### PR DESCRIPTION
**Pull request by matej494**
Added function: Present 'MovieDetailsViewController' when 'Movie' is tapped in 'MovieSearchViewController'.

### CODE REVIEW

File: Movieator/App/Movie search/MovieSearchViewControllerDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+//  Movieator
+//
+//  Created by Matej Korman on 23/05/2018.
+//  Copyright © 2018 Codeopolius. All rights reserved.
+//
+
+import Foundation
+
+protocol MovieSearchViewControllerDelegate {
+    func goToMovieDetails(forMovie movie: Movie)
```
[the remark this answers is not in the document]
Comment (matej494) [reply Movieator/App/Movie search/MovieSearchViewControllerDelegate.swift]: 'didSelectMovie(movie: Movie)' or 'didSelectItem(movie: Movie)'.
@dkulundzic

[the remark this answers is not in the document]
Comment (dkulundzic) [reply Movieator/App/Movie search/MovieSearchViewControllerDelegate.swift]: Lets follow Apple's convention when defining delegate methods:
`movieSearch(_ movieSearch: MovieSearchViewController, didSelectMovie movie: Movie)`.

**Comment by dkulundzic** [comment]
Looks good!

**Comment by matej494** [comment]
Merged branch 'search_movies' into branch 'develop' without conflicts.

**Pull request merged**